pandas/tests/test_window.py

```diff
@@ -1358,9 +1390,10 @@ def get_result(arr, window, min_periods=None, center=False):
             assert notna(result[4])
 
             # min_periods=0
-            result0 = get_result(arr, 20, min_periods=0)
```

Comment:
I had to disable this subset of tests, since I don't think it's true that `min_period=0` is the same as `min_period=1`.

Reply:
so make new ones that make this diff clear (i agree they should be different).

Reply:
Done, added a keyword to control this check, since it is true for most operations. `sum` is tested up above in `test_missing_minp_zero`.